setup.py

```diff
@@ -33,11 +38,19 @@ def get_version():
 VERSION = get_version()
 
 if __name__ == "__main__":
+    version.check_module_dependencies(manual_install_only=True)
+
     old_path = os.getcwd()
     local_path = os.path.dirname(os.path.abspath(sys.argv[0]))
     os.chdir(local_path)
     sys.path.insert(0, local_path)
 
+    # Convert module metadata into list of {mod_name}>={minver}
+    #   as required by install_requires
+    formatted_dependencies = ['%s>=%s' % (mod, meta['minver'])
```

Comment:
One nice thing about the current setup is that there's only one place where the dependencies are defined (in 'version.py'); the needed strings to pass to `setup` are created from that data here.